internal/core/processing.go

```diff
@@ -121,15 +123,14 @@ func ProcessBookmark(req ProcessRequest) (book model.Bookmark, isFatalErr bool,
 	// Save article image to local disk
 	for i, imageURL := range imageURLs {
 		err = DownloadBookImage(imageURL, imgPath)
-		if err != nil {
-			if err.Error() == fmt.Sprintf("%s is not a supported image", imageURL) {
-				log.Printf("Not found image for URL: %s", imageURL)
-				if i == len(imageURLs)-1 {
-					os.Remove(imgPath)
-				}
-			} else {
-				log.Printf("File download not successful for image URL: %s", imageURL)
+		if err != nil && errors.Is(err, ErrNoSupportedImageType) {
+			log.Printf("Not found image for URL: %s", imageURL)
```

Comment:
Let's print the error with the URL for now, to be consistent.

```suggestion
			log.Printf("%s: %s", err.String(), imageURL)
```